packages/xo-web/src/xo-app/dashboard/health/unhealthyVdis.js

```diff
@@ -49,17 +49,52 @@ const COLUMNS = [
   },
 ]
 
+const COLUMNS_VDI_UNKNOWN_PARENT = [
+  {
+    itemRenderer: srId => <Sr id={srId} link />,
+    name: _('sr'),
+    sortCriteria: 'name_label',
+  },
+  {
+    itemRenderer: (srId, { chainsLengthBySr }) => (
+      <div>
+        {map(chainsLengthBySr[srId].vdisWithUnknownVhdParent, vdiId => (
+          <Vdi id={vdiId} />
+        ))}
+      </div>
+    ),
+    name: _('vdisWithInvalidVhdParent'),
+  },
+]
+
 const UnhealthyVdis = decorate([
   addSubscriptions({
-    unhealthyVdiChainsLengthBySr: subscribeSrsUnhealthyVdiChainsLength,
+    chainsLengthBySr: subscribeSrsUnhealthyVdiChainsLength,
```

Comment:
Why?

Reply:
`unhealthyChainBySr`?  because it will contain  `vdisWithUnknownVhdParent`

Reply:
It actually contains 2 things about those VDIs now, so maybe something like `unhealthyVdisInfo`? Or `vdisHealth`? Or at least something that makes sense when you access a property of it because I personally don't like `chainsLengthBySr[srId].unhealthyVdis` since I'd expect `chainsLengthBySr[srId]` to be a length and accessing a property of a length doesn't really make sense.